### ISSUE TITLE
make all-images fails

### ISSUE THREAD

**Issue by colonelfazackerley** [andybalaam/rabbit-escape#571]
```
git clone https://github.com/andybalaam/rabbit-escape.git
cd rabbit-escape
make all-images
<fails>
make compile
make all-images
<completes>
```

output on fail
```
.. Converting all 64x64 images SVG->PNG
./build-scripts/bulk-convert-images images-src rabbit-escape-ui-swing/src/rabbitescape/ui/swing/images64 192 > /dev/null

** (inkscape:9681): WARNING **: 20:14:28.900: File path "rabbit-escape-ui-swing/src/rabbitescape/ui/swing/images64/rabbit_climb_left_continue_04.png" includes directory that doesn't exist.

make: *** [Makefile:350: all-images] Error 1
```

**Comment by andybalaam** [comment]
Easy to reproduce on Ubuntu too:

```bash
andrebal@andreballx:~$ cd /tmp/
andrebal@andreballx:/tmp$ git clone git@github.com:andybalaam/rabbit-escape.git
Cloning into 'rabbit-escape'...
remote: Counting objects: 25466, done.
remote: Compressing objects: 100% (12/12), done.
remote: Total 25466 (delta 2), reused 10 (delta 2), pack-reused 25451
Receiving objects: 100% (25466/25466), 49.73 MiB | 4.24 MiB/s, done.
Resolving deltas: 100% (15257/15257), done.
andrebal@andreballx:/tmp$ cd rabbit-escape/
andrebal@andreballx:/tmp/rabbit-escape$ make all-images
.. Making rabbot image from rabbit one: bin/imagessvggen/rabbot_rising_right_end_03.svg
SNIP
.. Making rabbot image from rabbit one: bin/imagessvggen/rabbot_brollychuting_02.svg
.. Converting all 32x32 images SVG->PNG
./build-scripts/bulk-convert-images images-src rabbit-escape-ui-swing/src/rabbitescape/ui/swing/images32 96      > /dev/null

** (inkscape:25927): WARNING **: 09:33:33.420: File path "rabbit-escape-ui-swing/src/rabbitescape/ui/swing/images32/rabbit_bridging_right_06.png" includes directory that doesn't exist.

Makefile:345: recipe for target 'all-images' failed
make: *** [all-images] Error 1
```

**Comment by andybalaam** [comment]
Fixed in e5e7f312f2e954035dd617948ef0565ffc7e5de4
